.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+# CircleCI configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.2.0  # Enables Windows executors
+
+jobs:
+  
+  linux-python-38: &linux-template
+    docker:
+      - image: circleci/python:3.8
+
+    environment:
+      PYTHON: python
+
+    working_directory: ~/repo
+
+    steps:
+
+      - checkout
+      - run:
+          name: install dependencies
+          command: |
+            export PATH=$PATH:$HOME/.local/bin
```

Comment:
Replace these commands with the commands from "staging" below. Then remove "staging." This set of commands is for building and testing the signac package from source, so it's not useful here.